src/main/webapp/app/core/course/manage/course-management-container/course-management-container.component.ts

```diff
@@ -136,11 +138,16 @@ export class CourseManagementContainerComponent extends BaseCourseContainerCompo
         this.subscription = this.route.firstChild?.params.subscribe((params: { courseId: string }) => {
             const id = Number(params.courseId);
             this.handleCourseIdChange(id);
+            this.checkIfOverviewPage();
+        });
+        this.urlSubscription = this.router.events.pipe(filter((event) => event instanceof NavigationEnd)).subscribe(() => {
+            this.checkIfOverviewPage();
```

Comment:
We could also do this with a computed signal, but as it does not really make a difference here I do not mind this solution